upstairs/src/lib.rs

```diff
@@ -5045,6 +5087,10 @@ pub enum IOop {
         dependencies: Vec<u64>, // Jobs that must finish before this
         writes: Vec<crucible_protocol::Write>,
     },
+    ReadFill {
+        dependencies: Vec<u64>, // Jobs that must finish before this
```

Comment:
also missing read_requests?

Reply:
ignore this - we discussed my confusion here, and this isn't a Read

Reply:
Rename and no need for `ReadRequests` here